src/packages/DAO/elections.dao.ts

```diff
@@ -53,6 +53,23 @@ export async function startElection(formData: FormData) {
   revalidatePath("/");
 }
 
+const VALUES = ["NOT_STARTED", "ONGOING", "FINISHED"] as const;
+
+export async function addElection(formData: FormData) {
```

Comment:
Lest use names for methods that resemble HTTP verbs when we can

```suggestion
export async function createElection(formData: FormData) {
```

CREATE, UPDATE, PATCH, DELETE. Upsert is also common